upstash_vector/core/index_operations.py

```diff
@@ -260,6 +261,73 @@ def query_many(
             for query_result in result
         ]
 
+    def resumable_query(
+        self,
+        vector: Optional[Union[List[float], SupportsToList]] = None,
+        top_k: int = 10,
+        include_vectors: bool = False,
+        include_metadata: bool = False,
+        filter: str = "",
+        data: Optional[str] = None,
+        namespace: str = DEFAULT_NAMESPACE,
+        include_data: bool = False,
+        max_idle: int = 3600,
+    ) -> ResumableQuery:
+        """
+        Creates a resumable query.
```

Comment:
Can we add a sentence about the necessity of calling stop() on resumable queries? It would be helpfull